### PR TITLE
S04 issue99/105 header admin

### PR DESCRIPTION
**Pull request by val-fr69**
Implementation of header customization from the admin part + new css adapted to the latest feature implemented

### CODE REVIEW

**Review by maxdlr — APPROVED** [review]

**Review by l00ma — APPROVED** [review]

File: templates/home/index.html.twig [hunk 1]
```diff
@@ -14,68 +14,89 @@
...
-									<span class="">REGARDER</span>
-									<span class="fs-2 ms-2 mt-1">{{ playFill|raw }}</span>
-								</a>
-							</div>
-
-							<div class="header-infos fs-5 z-1 d-flex d-sm-none title-font text-dark col-6 col-lg-3 justify-content-start align-items-end">
-								<div>{{headerVideo[0].title|u.truncate(30, '...')}}</div>
-							</div>
-						</div>
+<<<<<<< HEAD
```
Comment (maxdlr) [inline templates/home/index.html.twig:36]: A supprimer

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -14,68 +14,89 @@
...
+
+		{{ include('shared/gallery/bt-gallery.html.twig', {
+        title: 'Les plus vues',
+        subtitle: false,
+        count: 4,
+        contents: latestVideos,
+        id: 'mostViewed'
+    }) }}
+
+=======
```
Comment (maxdlr) [inline templates/home/index.html.twig:99]: Pareil

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -111,6 +132,7 @@
 				}) }}
 
 		</section>
+>>>>>>> c3ec14ee8c46161ac1b9832dbe85840422b0e27c
```
Comment (maxdlr) [inline templates/home/index.html.twig:135]: Pareil

**Review by maxdlr — CHANGES_REQUESTED** [review]

File: templates/home/index.html.twig (hunk 1 above)
Comment (aldaitzwild) [inline templates/home/index.html.twig:36]: Unresolved conflict

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by maxdlr — APPROVED** [review]

**Review by l00ma — APPROVED** [review]

**Pull request merged**